ui.html

```diff
@@ -0,0 +1,77 @@
+<html>
+<style>
+  body {
+    font: 10px sans-serif;
+    text-align: center;
+    margin: 10px;
+  }
+
+  button {
+    border-radius: 5px;
+    background: white;
+    color: black;
+    border: none;
+    padding: 8px 12px;
+    margin: 0 2px;
+    box-shadow: inset 0 0 0 1px black;
+    outline: none;
+  }
+
+  button:active {
+    background: lightgray;
+  }
+</style>
+
+<button id="start" style="width: 70;">Start</button>
+<button id="reset">Reset</button>
+<button id="help">?</button>
+<b id="helptext" hidden> <br /> <br /> Type the time you want to count down in "Timer: HH:MM:SS" format in any text
+  field (e.g. Timer: 5:00) then press 'start' </b>
+
```

Comment:
Maybe we should also add a hint that selecting that doesn't start with "Timer: " also works